fgpyo/io/__init__.py

```diff
@@ -14,30 +14,49 @@
 ## fgpyo.io Examples:
 
 ```python
+>>> import fgpyo.io as fio
+>>> from pathlib import Path
+
+```
+
+Assert that a path exists and is readable
+
+```python
+>>> tmp_dir = Path(getfixture("tmp_path"))
```

Comment:
where is `getfixture` defined?

Reply:
https://docs.pytest.org/en/7.1.x/how-to/doctest.html#using-fixtures